enhancements/ingress/operator-resource-configuration.md

```diff
@@ -0,0 +1,900 @@
+9. Platform administrator confirms QoS class with `oc get pod -n openshift-ingress -o jsonpath='{.items[*].status.qosClass}'`
+
+### API Extensions
+
+Add a new field to the existing v1 IngressController API in the `operator.openshift.io` 
+group, gated behind a feature gate. This approach is preferred by the networking team 
+for its simplicity - adding the feature directly to the stable v1 API while protecting 
+it behind a feature gate during the Tech Preview period.
+
+The new field allows configuring resource **limits** for router pods. The existing v1 API's 
+`spec.nodePlacement.resources` field currently allows setting requests, but this 
+enhancement adds a new field to also set limits, enabling router pods to achieve 
+Guaranteed QoS class.
+
+#### Feature Gate
+
+This feature requires a new feature gate: **`IngressRouterResourceLimits`**
+
+The feature gate controls whether the new v1 API field is recognized and enforced 
+by the ingress-operator. Initially, the feature gate will be part of the 
+TechPreviewNoUpgrade feature set, and will be promoted to the Default feature set 
+once the feature graduates to GA.
+
+**Enabling the Feature Gate:**
+
+To enable the feature gate in your OpenShift cluster, you can use either the patch command 
+or apply a FeatureGate configuration.
+
+**Option 1: Enable all Tech Preview features (includes IngressRouterResourceLimits):**
+```bash
+oc patch featuregate cluster --type merge --patch '{"spec":{"featureSet":"TechPreviewNoUpgrade"}}'
+```
+
+**Option 2: Enable only the specific feature gate using patch command:**
+```bash
+oc patch featuregate cluster --type merge --patch '{"spec":{"featureSet":"CustomNoUpgrade","customNoUpgrade":{"enabled":["IngressRouterResourceLimits"]}}}'
+```
+
+**Option 3: Apply a custom FeatureGate configuration file:**
+```yaml
+apiVersion: config.openshift.io/v1
+kind: FeatureGate
+metadata:
+  name: cluster
+spec:
+  featureSet: CustomNoUpgrade
+  customNoUpgrade:
+    enabled:
+    - IngressRouterResourceLimits
+```
+
+Apply with:
+```bash
+oc apply -f featuregate.yaml
+```
+
+**Note**: Enabling feature gates may require cluster components to restart. For 
+production environments, test in non-production clusters first. Using `TechPreviewNoUpgrade` 
+or `CustomNoUpgrade` means the cluster cannot be upgraded and should only be used for 
+testing.
+
+#### API Changes
+
+Add a new field to the existing `IngressControllerSpec` in the v1 API:
+
+```go
+package v1
+
+import (
+    metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+    corev1 "k8s.io/api/core/v1"
+)
+
+// IngressControllerSpec is the specification of the desired behavior of the IngressController.
+type IngressControllerSpec struct {
+    // ... existing v1 fields ...
+
+    // tuning defines parameters for tuning the performance of ingress controller pods.
+    // +optional
+    Tuning *IngressControllerTuning `json:"tuning,omitempty"`
```

Comment:
Is this IngressControllerTuningOptions https://github.com/openshift/api/blob/bfa868a224015e94456731c1b5b0c849f901b417/operator/v1/types_ingress.go#L263 or a new field?